firestore/query.go

```diff
@@ -43,6 +43,7 @@ type Query struct {
 	orders                 []order
 	offset                 int32
 	limit                  *wrappers.Int32Value
+	limitToLast            *wrappers.Int32Value
```

Comment:
I'm not sure, but it may be better to replace it with the indicator points to the order with `iota`.
WDYT?

Reply:
I remember why I used `limitToLast` as an integer value. In `Get()` we're calling `Documents()` method. `Documents()` is a stream method, it can't use `limitToLast`. So, if we'll be using limit type (instread of integer value) then in `Get()` method body, after flipping the orders and cursors, but before `Documents()` call, we'll have to change limit type to default (limit from the start), or we'll get an error that we can't use `limitToLast` in stream method. I've decided that it's more clear to use an integer than changing the type. Still, let's think about it little bit more, maybe there is a better way...